main.go

```diff
@@ -1,19 +1,21 @@
 package bt
 
-import "bytes"
-import "net/url"
-import "net/http"
-import crypto_rand "crypto/rand"
-import "encoding/json"
-import "fmt"
-import "io/ioutil"
-import math_rand "math/rand"
-import "os"
-import "regexp"
-import "runtime"
-import "strconv"
-import "strings"
-import "time"
+import (
+	"bytes"
+	"net/http"
+	"net/url"
+
+	crypto_rand "crypto/rand"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	math_rand "math/rand"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+)
```

Comment:
@MateuszKepczynskiSauce I didn't touch that part, this is how `gofmt` formatted the code. Not sure if I can change that.

Reply:
hmm, weird, looks the formatter added newlines in between imports, not sure why. When I removed them manually it sorted the packages correctly. I've also removed the underscores so now we have more canonical `mathrand` and `cryptorand` instead of `math_rand` and `crypto_rand`.